release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+
+For additional instructions on upgrading, see the documentation in the "Upgrade documentation" section. Procedures for upgrading by using Helm and `roxctl` now include a step to back up the Central database before upgrading Central.
+
+[id="upgrade-acscs-rn40"]
+=== Upgrading {rh-rhacscs-first} to version 4.0
+
+To upgrade, {product-title-managed-short} you must switch the Operator channel to `stable` for your secured clusters.
+
+[id="upgrade-documentation-rn40"]
+=== Upgrade documentation
+
+For more information about upgrading to release 4.0, see the following documentation:
+
+- xref:../upgrading/upgrade-operator.adoc[Upgrading by using the Operator]
+- xref:../upgrading/upgrade-helm.adoc[Upgrading using Helm charts]
+- xref:../upgrading/upgrade-roxctl.adoc[Manually upgrading using the roxctl CLI]
+
+[id="external-postgresql-database-rn40"]
+=== Installing Central with an external PostgreSQL database (Technology Preview)
+
+With this release, you can test the use of your existing PostgreSQL infrastructure to provision a database for {product-title-short}.
+
+:FeatureName: External PostgreSQL database
+include::snippets/technology-preview.adoc[]
+
+{product-title-short} 4.0 includes this feature as a Technology Preview and it is not supported for production systems. It currently has several limitations, including the need for privileged superuser access. We are seeking customer feedback and testing.
+
+For more information, see xref:../installing/installing_ocp/install-central-ocp.adoc#install-central-operator-tech-preview_install-central-ocp[Installing Central with an external database using the Operator method].
+
+[id="acs-cloud-release-rn40"]
+=== {rh-rhacscs-first} Limited Availability
+
+{product-title-managed-short} provides a fully hosted and managed Central instance that allows you to secure your on-premise and cloud-based Kubernetes clusters.
+
+{product-title-managed-short} is available for approved customers. Customers can get {product-title-managed-short} by using the Amazon Web Services (AWS) Marketplace or from their regular Red Hat channels. For more information, contact Red Hat Sales.
+
+During the Limited Availability phase, Red Hat will provide full support with committed service level agreements (SLAs) and response times. We will also continue to provide an email list for feedback and general questions. The following items apply to {product-title-managed-short} limited availability:
+
+- {product-title-managed-short} provides the capability to access repositories local to the secured clusters. This functionality is supported only for OpenShift secured clusters and Red Hat Quay that will have local scanning capabilities for internal and extended registries in the customer environments.
+- The Ireland (eu-west-1) region is now available when creating an {product-title-managed-short} instance. Red Hat will ensure all data residency requirements for data are met per the EU standards.
```

Comment:
Maybe mark up `eu-west-1`? Unless you don't elsewhere already for references to region names.